test/e2e/scheduling/preemption.go

```diff
@@ -422,12 +422,28 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 		node, err = cs.CoreV1().Nodes().UpdateStatus(nodeCopy)
 		framework.ExpectNoError(err)
 
+		// TODO(Huang-Wei): remove this after #74626 is resolved
+		// print out existing priorities
+		priorityList, err := cs.SchedulingV1().PriorityClasses().List(metav1.ListOptions{})
+		if err != nil {
+			framework.Logf("Listing all priorities: %v", err)
```

Comment:
Should this log message be reworded to "Unable to list priorities"?

Reply:
Sure.